docs/quest_template.md

```diff
@@ -8,73 +8,219 @@ Contains all basic definitions of available quests.
+| [AreaDescription][78]           | text         |           |     | YES  |         |       |         |
+| [QuestCompletionLog][79]        | text         |           |     | YES  |         |       |         |
+| [RequiredNpcOrGo1][80]          | mediumint(9) |           |     | NO   | 0       |       |         |
+| [RequiredNpcOrGo2][81]          | mediumint(9) |           |     | NO   | 0       |       |         |
+| [RequiredNpcOrGo3][82]          | mediumint(9) |           |     | NO   | 0       |       |         |
+| [RequiredNpcOrGo4][83]          | mediumint(9) |           |     | NO   | 0       |       |         |
+| [RequiredNpcOrGoCount1][84]     | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredNpcOrGoCount2][85]     | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredNpcOrGoCount3][86]     | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredNpcOrGoCount4][87]     | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemId1][88]           | mediumint(8) | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemId2][89]           | mediumint(8) | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemId3][90]           | mediumint(8) | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemId4][91]           | mediumint(8) | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemId5][92]           | mediumint(8) | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemId6][93]           | mediumint(8) | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemCount1][94]        | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemCount2][95]        | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemCount3][96]        | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemCount4][97]        | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemCount5][98]        | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [RequiredItemCount6][99]        | smallint(5)  | unsigned  |     | NO   | 0       |       |         |
+| [Unknown0][100]                 | tinyint(3)   | unsigned  |     | NO   | 0       |       |         |
+| [ObjectiveText1][101]           | text         |           |     | YES  |         |       |         |
+| [ObjectiveText2][102]           | text         |           |     | YES  |         |       |         |
+| [ObjectiveText3][103]           | text         |           |     | YES  |         |       |         |
+| [ObjectiveText4][104]           | text         |           |     | YES  |         |       |         |
+| [VerifiedBuild][105]            | smallint(6)  |           |     | YES  | 0       |       |         |
```

Comment:
Wrong.

Attributes and Type should be capitalized, no width for integers.

Reply:
Corrected.